src/styles/styles.css

```diff
@@ -36,26 +38,68 @@ body {
 .ads-container {
     display: grid;
     gap: 1rem; /* Adjust gap as needed */
-    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr)); /* Adjust minmax value based on desired minimum size */
+    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr)); /* Ensures flexibility */
 }
 
 /* 3x3 grid for screens wider than 1024px */
 @media (min-width: 1024px) {
     .ads-container {
-        grid-template-columns: repeat(3, 1fr);
+        grid-template-columns: repeat(3, 1fr); /* 3 columns */
     }
 }
 
 /* 2x3 grid for screens between 600px and 1023px */
 @media (min-width: 600px) and (max-width: 1023px) {
     .ads-container {
-        grid-template-columns: repeat(2, 1fr);
+        grid-template-columns: repeat(2, 1fr); /* 2 columns */
     }
 }
 
 /* 1x3 grid for screens under 600px */
 @media (max-width: 599px) {
     .ads-container {
-        grid-template-columns: repeat(1, 1fr);
+        grid-template-columns: repeat(1, 1fr); /* 1 column */
     }
-}
\ No newline at end of file
+}
+
+
+/* For horizontal ads (16:9 aspect ratio) */
```

Comment:
this is aspect-ratio logic, where if the image gets formalized as 16:9 or 9:16 ratio